src/routes/add-place.ts

```diff
@@ -31,6 +31,19 @@ export default async function addPlace(fastify: FastifyInstance) {
     return resp.view('src/liquid/app/view.html', tmplData);
   });
 
+  fastify.post('/widget', async (req, resp) => {
+    const queryParams: any = req.query;
+    const contactType = Config.getContactType(queryParams.place_type).contact_properties.find(prop => prop.type === queryParams.input);
+    return resp.view('src/liquid/components/contact_type_property.html', {
+      data: req.body,
+      include: {
+        prefix: 'contact_',
+        place_type: queryParams.place_type,
```

Comment:
`const { place_type, input } = queryParam;`

This quick alias can be useful to increase code readability - reducing line length, making refactoring easier, etc